includes/js/chapman.js

```diff
@@ -0,0 +1,558 @@
+/* $Id: chapman.js,v 1.1 2013/01/30 10:02:28 bbannon Exp $ */
```

Comment:
I was mostly planning on using it for the Date functions.
Also, it has the scrollers from v_h_scrolls.js incorporated.
There are a couple of other functions that looked interesting but, I don't have solid plans for them, yet. I put them in just in case.